src/EnergyPlus/Plant/PlantManager.cc

```diff
@@ -4610,6 +4619,81 @@ void CheckOngoingPlantWarnings(EnergyPlusData &state)
     }
 }
 
+void ReportPlantCompWaterFlowData(EnergyPlusData &state, bool const reportFlag)
+{
+    // Reporting will be 24 hrs of data by plant loop for each component connected to that loop. Then the next loop's components will be reported.
+    // Could be changed to report all loop's and components at the same time (i.e., as single group from 00:15 to 24:00 instead of multiple groups)
+    // sum equipment water flow rate time series
+    for (int LoopNum = 1; LoopNum <= state.dataHVACGlobal->NumPlantLoops; ++LoopNum) {
+        if (state.dataPlnt->PlantLoop(LoopNum).compDesWaterFlowRate.size() == 0) {
+            continue;
+        }
+        state.dataPlnt->PlantLoop(LoopNum).plantDesWaterFlowRate.resize(size_t(24 * state.dataGlobal->TimeStepsInHour));
+        for (size_t ts = 0; ts < size_t(24 * state.dataGlobal->TimeStepsInHour); ++ts) {
+            state.dataPlnt->PlantLoop(LoopNum).plantDesWaterFlowRate[ts] = 0.0;
+            for (auto &thisCoil : state.dataPlnt->PlantLoop(LoopNum).compDesWaterFlowRate) {
+                state.dataPlnt->PlantLoop(LoopNum).plantDesWaterFlowRate[ts] += thisCoil.tsDesWaterFlowRate[ts + 1];
+            }
+        }
+    }
```

Comment:
Above sums plant component water flow rates into the master plantDesWaterFlowRate array.